core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -754,17 +754,41 @@ class AutoTuner(
       limitedLogicList: Option[Seq[String]] = Some(Seq("spark.sql.shuffle.partitions")),
       showOnlyUpdatedProps: Boolean = true):
       (Seq[RecommendedPropertyResult], Seq[RecommendedCommentResult]) = {
-    filterByUpdatedPropertiesEnabled = showOnlyUpdatedProps
-    limitedLogicList.foreach { limitedSeq =>
-      limitedSeq.foreach(_ => limitedLogicRecommendations.add(_))
-    }
-    skipList.foreach(skipSeq => skipSeq.foreach(_ => skippedRecommendations.add(_)))
-    if (processPropsAndCheck) {
-      initRecommendations()
-      calculateRecommendations()
+      val isPluginLoaded = getPropertyValue("spark.plugins") match {
+        case Some(f) => f.contains("com.nvidia.spark.SQLPlugin")
+        case None => false
+      }
+      val rapidsEnabled = getPropertyValue("spark.rapids.sql.enabled") match {
+        case Some(f) => f.toBoolean
+        case None => true
+      }
+    if (!isPluginLoaded || !rapidsEnabled) {
+      appendComment("AutoTuner recommendations only support " +
+          "eventlogs generated by Spark applications\n  running on GPU hardware with Spark " +
+          "RAPIDS Plugin enabled.")
+      if (!isPluginLoaded) {
+        appendComment("The Plugin jar is missing. Please refer to https://github.com/NVIDIA/" +
```

Comment:
> `The Plugin jar is missing`

The checks cannot accurately determine that . This could confuse the user.
I suggest rephrasing it to indicate that the the RAPIDS was not added to the spark plugins